fabric-item-api-v1/src/main/java/net/fabricmc/fabric/mixin/item/ItemEntityMixin.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.item;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+@Mixin(ItemEntity.class)
+public abstract class ItemEntityMixin extends Entity {
+
+	@Shadow
+	private int itemAge;
+
+	private ItemEntityMixin(EntityType<?> type, World world) {
+		super(type, world);
+	}
+
+	@Inject(at = @At("HEAD"), method = "setDespawnImmediately")
+	private void modifyDespawnImmediately(CallbackInfo ci) {
```

Comment:
Im not sure this is going to work, as `setDespawnImmediately` sets the item age to `itemAge = 5999` after this mixin.